devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -169,11 +189,37 @@ def doConfigure() {
 	if (isSchlageLock()) {
 		cmds << secure(zwave.configurationV2.configurationGet(parameterNumber: getSchlageLockParam().codeLength.number))
 	}
-	cmds = delayBetween(cmds, 4200)
+	cmds << setClock()
+	cmds << configureScheduling()
+	// delayBetween does the flattening
+	cmds = delayBetween(cmds, 200)
```

Comment:
Just pointing out that we've had issues elsewhere with short delays between lock commands. I'd prefer that this value at least remained `4200`